aquacomputer_d5next.c

```diff
@@ -498,7 +530,10 @@ static int aqc_read_string(struct device *dev, enum hwmon_sensor_types type, u32
 
 	switch (type) {
 	case hwmon_temp:
-		*str = priv->temp_label[channel];
+		if (channel < priv->num_temp_sensors)
+			*str = priv->temp_label[channel];
+		else
+			*str = priv->virtual_temp_label[channel - priv->num_temp_sensors - 1];
```

Comment:
remove  `-1` as channel starts at 0 and so the first virtual sensor would get virtual_temp_label[-1]